languages/indonesian/apk/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="english_app_name">English for AnySoftKeyboard</string>
```

Comment:
this should be:
```
<string name="indonesian_app_name">Indonesian for AnySoftKeyboard</string>
```

Reply:
Oh, I missed this. It didn't mentioned in new steps. Another time I  should use silversearcher to find all string that need to be changed.